src/main/java/com/swrve/ratelimitedlogger/LogWithPatternAndLevel.java

```diff
@@ -136,7 +134,7 @@ private void reportSuppression(long whenLimited) {
         if (numSuppressed == 0) {
             return;  // special case: we hit the rate limit, but did not actually exceed it -- nothing got suppressed, so there's no need to log
         }
-        Duration howLong = new Duration(whenLimited, elapsedMsecs());
+        Duration howLong = Duration.ofMillis(whenLimited).plusMillis(elapsedMsecs());
```

Comment:
I'm not sure this is correct.  The original is using this API: http://joda-time.sourceforge.net/apidocs/org/joda/time/Duration.html#Duration(long, long)

Duration(long startInstant, long endInstant) 
          Creates a duration from the given interval endpoints.

Reply:
You're right, made a mistake here, I need to reverse it to Duration howLong = Duration.ofMillis(elapsedMsecs()).minusMillis(whenLimited);